### DISCUSSION TITLE
H200 Hub and C420 Camera integration with HA?

### DISCUSSION THREAD

**Discussion by Darthmonkey** [JurajNyiri/HomeAssistant-Tapo-Control#704] [Q&A] [answered]
Can someone please definitively answer the question whether it's currently (13 Nov 24) possible to view C420 battery cameras on Home Assistant via an H200 hub?

I've seen a lot of posts suggesting it's not possible yet, but it may be a feature that Netgear allow, or is reverse-engineered. The main issue being that a constant stream is not provided as it'd drain the battery.

I've also seen a lot of posts asking why their C420 camera has stopped working, suggesting that it was once working. These posts don't say they're battery cameras, or that they're using an H200 hub, I've made the assumption that all c420 cameras are battery powered and/or don't have an available IP address for HA.

I've successfully linked my H200 hub to Matter, and installed HomeAssistant-Tapo-Control, but no dice so far.

**Comment by phardy2410** [comment]
if you cant enable a camera account on the camera then you cant get the stream into HA. I mean I'm sure someone has figured it out but not easily. Tapo elected to prtect battery life and ban camera accounts on battery powered devices it seems, my D235 doorbell is the same and i dont really want to extract the wires from inside a recently fitted door frame ya know. wife factor approval zero

**Reply by JurajNyiri** [reply]
This is incorrect. Since version 7.0.0 you can use direct streams.

**Comment by phardy2410** [comment]
im considering (strongly) the Hub 500 but i bet a similar, limited experience will exist as far as live streams from battery powered devices (only the doorbell for me) the rest of my cameras are mains powered

**Comment by JurajNyiri** [comment, marked as answer]
Direct streams are supported since https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/7.0.0 .
